annoying/tests/urls.py

```diff
@@ -1,13 +1,13 @@
 """URLs for django-annoying's tests"""
 from __future__ import absolute_import
 
-from django.conf.urls import patterns
+from django.conf.urls import url
```

Comment:
This would be pretty easy to make backwards-compatible like so:

```
try:
    from django.conf.urls import patterns
except ImportError:
    # Hack for backwards-compatibility.
    patterns = lambda x: list(x[1:])

urlpatterns = patterns('',
    <etc>
```